### PR TITLE
Add MQTT settings info to Tasmota page

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Add MQTT settings info to Tasmota page

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- This PR fixes or closes issue: https://github.com/home-assistant/home-assistant.io/issues/17152

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by emontnemery** [comment]
@cweinhofer please help to review the changes

**Review by cweinhofer — DISMISSED** [review]

**Comment by cweinhofer** [comment]
Looks good to me. Thanks!

File: source/_integrations/tasmota.markdown [hunk 1]
```diff
@@ -27,7 +27,8 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
 
 ## Requirements
 
-- MQTT server and the [MQTT integration](/integrations/mqtt/) set up in Home Assistant.
+- MQTT broker and the [MQTT integration](/integrations/mqtt/) set up in Home Assistant.
+- Tasmota devices' MQTT setting configured to communicate with the MQTT broker
```
Comment (frenck) [inline source/_integrations/tasmota.markdown]:
```suggestion
- Tasmota devices' MQTT setting configured to communicate with the MQTT broker.
```

File: source/_integrations/tasmota.markdown [hunk 2]
```diff
@@ -46,3 +47,7 @@ Tasmota Buttons, Fans, Lights, relays, Sensors, Shutters and Switches are suppor
   ![iot](https://user-images.githubusercontent.com/7702766/99080146-a1d43980-259f-11eb-856b-addb53695381.png)
 
 {% include integrations/config_flow.md %}
+
+You must also configure each Tasmota device's MQTT settings to communicate with whatever MQTT broker you are using. Enter the broker address under host and enter a username / password combination that allows access to the broker.
```
Comment (frenck) [inline source/_integrations/tasmota.markdown]:
```suggestion
You must also configure each Tasmota device's MQTT settings to communicate with whatever MQTT broker you are using. Enter the broker address under host and enter a username/password combination that allows access to the broker.
```

File: source/_integrations/tasmota.markdown [hunk 3]
```diff
@@ -46,3 +47,7 @@ Tasmota Buttons, Fans, Lights, relays, Sensors, Shutters and Switches are suppor
...
+
+If using the Mosquitto Broker Add-on for Home Assistant, first create a new Home Assistant user under Configuration > Users. Then, in Tasmota > Configure MQTT under "Host" enter the address of your Home Assistant instance and under "User" and "Password" enter the Home Assistant user you just created.
```
Comment (frenck) [inline source/_integrations/tasmota.markdown]:
```suggestion
If using the Mosquitto Broker add-on for Home Assistant, first create a new Home Assistant user under {% my users title="Configuration > Users" %}. Then, in Tasmota > Configure MQTT under "Host" enter the address of your Home Assistant instance and under "User" and "Password" enter the Home Assistant user you just created.
```

**Review by frenck — APPROVED** [review]
Thanks @emontnemery 👍

**Pull request merged**